qemu/tests/hpt_max_page_size.py

```diff
@@ -38,3 +39,13 @@ def _check_meminfo(key):
     if _check_meminfo("Hugepagesize") != params["expected_value"]:
         test.fail("The hugepage size doesn't match, "
                   "please check meminfo: %s " % _check_meminfo("Huge"))
+    # Please set 1G huge page as default huge page size on power9
+    if params.get("sub_type") == "hugepage_reset":
+        origin_nr = params.get("origin_nr", "1")
```

Comment:
if memory.get_huge_page_size() != int(params.get(expected_hugepage_size)
    test.cancel("......")

Reply:
Please refer to my previous comments. "It's product's behavior, should be "expected_value = 16384  but not expected_hugepage_size = 1048576 in P8 guest". It's proper for us to set it as "<" in my opinions, besides that, it should be the same issue to the first comment. Thanks, will update it soon.

Reply:
Updated